Homogeneous_kernel/include/CGAL/Homogeneous/function_objects.h

```diff
@@ -770,6 +770,64 @@ namespace HomogeneousKernelFunctors {
     }
   };
 
+    template <typename K>
+  class Compare_signed_distance_to_line_2
```

Comment:
Needs to be tested and documented in the Kernel concept... or just removed since it's not used by the PR.

Reply:
`Less_signed_distance_to_line_2`  now uses `Compare_..`.